notebooks/image_models/solutions/serving.ipynb

```diff
@@ -0,0 +1,1180 @@
+{
```

Comment:
<p>"and execute transfer learning quickly." &gt;&gt; "and use transfer learning to train quickly" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Thanks!</p>